packages/eslint-plugin/src/util/getWrappingFixer.ts

```diff
@@ -205,3 +215,21 @@ function isLeftHandSide(node: TSESTree.Node): boolean {
 
   return false;
 }
+
+/**
+ * Checks if a node's parent is arrow function expression and a inner node is object expression
+ */
+function isObjectExpressionInOneLineReturn(
+  node: TSESTree.Node,
+  innerNode: TSESTree.Node,
+): boolean {
+  if (
+    node.parent?.type === AST_NODE_TYPES.ArrowFunctionExpression &&
+    node.parent?.body === node &&
```

Comment:
[Nitpick] If `node.parent?.type` is something defined, we know `node.parent` exists. ✨ 

```suggestion
    node.parent.body === node &&
```

Reply:
Also -and this one is very optional, just posting in case you like it more, as I ever so slightly do 😄- you could simplify a bit to:

```ts
return (
  node.parent?.type === AST_NODE_TYPES.ArrowFunctionExpression &&
  node.parent.body === node &&
  innerNode.type === AST_NODE_TYPES.ObjectExpression
);
```